common/changes/office-ui-fabric-react/detailsListAltText_2018-11-22-00-05.json

```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "packageName": "office-ui-fabric-react",
+      "comment": "DetailsList: add alt text to img in example",
+      "type": "none"
```

Comment:
@cliffkoh are we good to use `type: "none"` changelogs? Asking since we had some issues with publishing a week or so ago.

Reply:
Yes it is fine :)

Reply:
Spoke offline and we're good here, unrelated.